### ISSUE TITLE
[Feature request]: A autoplay feature in `<Speech/>` component. 

### ISSUE THREAD

**Issue by Hetosk1** [SahilAggarwal2004/react-text-to-speech#7]
### Describe the feature you'd like to request

I really require a feature in which i can autoplay the text in the speech Component

### Describe the solution you'd like

Maybe a prop you can pass in the Speech component

### Describe alternatives you've considered

I new to this stuff.

**Comment by SahilAggarwal2004** [comment]
Hello @Hetosk1,

I completely agree with your idea of adding an autoplay feature to the `<Speech>` component, and I've also attempted to implement it in previous versions of `react-text-to-speech`.
Unfortunately, modern browsers restrict the use of the `SpeechSynthesis API` without user interaction. :(

For more information on this limitation, you can refer to the following sources:
- [MDN: SpeechSynthesisErrorEvent - not-allowed](https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisErrorEvent/error#not-allowed)
- [Chromium Developers Group](https://groups.google.com/a/chromium.org/g/blink-dev/c/WsnBm53M4Pc/m/B-5syvAvCAAJ)
- [Stack Overflow Discussion](https://stackoverflow.com/questions/54265423/javascript-speechsynthesis-speak-without-user-activation-is-no-longer-allowed)

**Comment by SahilAggarwal2004** [comment]
I could still implement it, and it may work on older browsers, but it would introduce inconsistencies that I don’t think are worth the effort. That said, I’ll keep exploring ways to bypass the user interaction requirement with some clever workaround.

If you’ve already discovered a hack to make this work, please let me know! 

Either way, thanks again for using `react-text-to-speech` and for giving me the opportunity to improve it further!

**Comment by SahilAggarwal2004** [comment]
Hi @Hetosk1,

I have some good news to share!

I discovered that while Chrome requires user interaction to initiate speech synthesis, Firefox does not have this restriction. Additionally, in Chrome, if a user interacts with the page (like by clicking on a link) or if the text property changes, speech will play automatically.

With this in mind, I've added an `autoPlay` prop to both the `useSpeech` hook and the `<Speech>` component in the latest version of react-text-to-speech (v0.19.0). This feature should enhance the user experience by allowing speech to play automatically.

Thank you for suggesting this valuable improvement!
